src/components/FilterBar.jsx

```diff
@@ -21,14 +30,52 @@ const amounts = {
   5000: '> 5000',
 };
 
-function FilterBar() {
+const sortingOptions = {
+  deadlineSoon: 'Deadline (Earliest to Latest)',
+  deadlineLatest: 'Deadline (Latest to Earliest)',
+  amountLow: 'Amount (Low to High)',
+  amountHigh: 'Amount (High to Low)',
+};
+
+export default function FilterBar(props) {
+  const { passedFunction } = props;
+
+  function modifySorting(sortOption) {
+    switch (sortOption) {
+      case 'one':
+        return passedFunction('deadline');
+      case 'two':
+        return passedFunction('deadlineReverse');
+      case 'three':
+        return passedFunction('amount.min');
+      case 'four':
+        return passedFunction('amount.max');
+      default:
+        return null;
+    }
+  }
```

Comment:
This does not do what you think it does. Besides the `deadline` option and sort of the `amount.min` functionality the other ones don't really work correctly.

Reply:
Punted for #342